### PR TITLE
Enhancement : Ec2 - Add describe-vpc-endpoint-services method support.

### PR DESCRIPTION
**Pull request by usmangani1**
Enhancement : Ec2 - Add describe-vpc-endpoint-services method support.
https://github.com/localstack/localstack/issues/2650

### CODE REVIEW

File: moto/ec2/models.py [hunk 1]
```diff
@@ -2995,6 +2995,29 @@ def create_vpc_endpoint(
...
 
+    def get_vpc_end_point_services(self):
+        vpc_end_point_services = self.vpc_end_points.values()
+
+        services = []
+        for value in vpc_end_point_services:
+            services.append(value.service_name)
+            print(".".join(reversed(value.service_name.split("."))))
+
+        availability_zones = [
```
Comment (bblommers) [inline moto/ec2/models.py:3006]: These don't have to be hardcoded:
https://github.com/spulec/moto/blob/2504a398f9ac9659986abaeb0ba19d6550eeb309/moto/ec2/models.py#L1811

**Comment by usmangani1** [comment]
> Thanks for the PR, @usmangani1! Got one minor niggle, happy to merge after that

@bblommers Not sure why this error is coming up here.Can you pelase help me out here ?

**Comment by usmangani1** [comment]
> > Thanks for the PR, @usmangani1! Got one minor niggle, happy to merge after that
> 
> @bblommers Not sure why this error is coming up here.Can you please help me out here ?

@bblommers Can you please help me get this merged ?

**Comment by bblommers** [comment]
That was an intermittent failure, @usmangani1 - I restarted the build, and it's fine now.

**Comment by bblommers** [comment]
This is now part of moto >= 1.3.15.dev1023

**Pull request merged**